federation/cluster/common.sh

```diff
@@ -65,11 +65,11 @@ function create-federation-api-objects {
     : "${FEDERATION_IMAGE_TAG?Must set FEDERATION_IMAGE_TAG env var}"
 
     export FEDERATION_APISERVER_DEPLOYMENT_NAME="federation-apiserver"
-    export FEDERATION_APISERVER_IMAGE_REPO="${FEDERATION_PUSH_REPO_BASE}/hyperkube"
+    export FEDERATION_APISERVER_IMAGE_REPO="${FEDERATION_PUSH_REPO_BASE}/hyperkube-amd64"
```

Comment:
This is somewhat subtle. In the previous/current mechanism, I used to build the hyperkube image on the fly and wouldn't prefix `-amd64`. Now that we are switching to using the real published images through our build system which uses the `name-<arch>` format we need this suffix. Also note, this doesn't hurt the existing system because of the change in line #297 below.